src/main/java/Duke.java

```diff
@@ -1,10 +1,175 @@
+import java.io.*;
+import java.util.*;
```

Comment:
Following our coding standard, imported classes should be listed explicitly. For example,
```suggestion
import java.io.BufferedReader;
```
I noticed the same issue in several other places as well.